cmd/main.go

```diff
@@ -108,6 +108,7 @@ func main() {
 			klog.ErrorS(err, "failed to get version")
 			klog.FlushAndExit(klog.ExitFlushTimeout, 1)
 		}
+		//nolint:forbidigo
```

Comment:
It's not clear why this comment is added unless you read through the PR discussion. Suggest adding that clarity directly in the comment so that future maintainers don't accidentally change this.